manifests/p/Piriform/CCleaner/ProTrial/6.23.0.11010/Piriform.CCleaner.ProTrial.installer.yaml

```
@@ -0,0 +1,31 @@
+# Automatically updated by the winget bot at 2024/Apr/17
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Piriform.CCleaner.ProTrial
+PackageVersion: 6.23.0.11010
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
+Scope: machine
+InstallModes:
+- interactive
+- silent
+UpgradeBehavior: install
+Commands:
+- ccleaner
+FileExtensions:
+- zzz
+AppsAndFeaturesEntries:
+- DisplayVersion: "6.21"

```

```suggestion
- DisplayVersion: "6.23"
```
Purely guessing here.